server/src/encrypt.py

```diff
@@ -0,0 +1,26 @@
+import jwt
+from cryptography.fernet import Fernet
+
+from src.settings import SECRET
+
+
+FERNET = Fernet(SECRET)
+
+
+def encrypt_password(msg: str) -> str:
+    enc_message = FERNET.encrypt(msg.encode())
+    return enc_message.decode('utf-8')
+
+
+def decrypt_password(msg: str) -> str:
+    msg = msg.encode('utf-8')
+    dec_message = FERNET.decrypt(msg).decode()
+    return dec_message
```

Comment:
Why you need to create extra variables?)
Function can be writed as 
return FERNET.decrypt(msg.encode('utf-8')).decode()